apps/04TowersOfHanoi.js

```diff
@@ -19,24 +19,64 @@ function printStacks() {
   console.log("c: " + stacks.c);
```

Comment:
Need to get in the habit of commenting. Even though you think you know exactly what is going on. It helps your though process and is a important practice in the real world, so we need to get used to it now